wled00/wled_server.cpp

```diff
@@ -9,6 +9,9 @@
 #ifndef WLED_DISABLE_PXMAGIC
   #include "html_pxmagic.h"
 #endif
+#ifndef WLED_DISABLE_GIFPLAYER
```

Comment:
Please wrap with a check for WLED_ENABLE_GIF is defined as no point having the player if you don't have GIF  support